software/raft/raft_server_riscv/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+cmake_minimum_required(VERSION 2.8)
+
+project(raft_server_riscv)
+
+add_executable(raft_server_riscv ../../../tests/crt.o ../../../tests/syscalls.o main.cc mica/util/cityhash/city_mod.cc)
+target_link_libraries(raft_server_riscv raft -T /home/$ENV{USER}/chipyard/tests/link.ld -nostartfiles -lgcc -mcmodel=medany)
```

Comment:
This assumes that the chipyard repo is always cloned into `$HOME`. Can we use a relative path instead like on the previous line? i.e. `../../../tests/link.ld`

Reply:
Agreed. That would be good.

Reply:
Unfortunately, just changing this to a relative path causes the linker to fail to find the script. There might be a fancier cmake trick I can do to get the path though.

Reply:
I fixed all of the other absolute paths I could find though.